packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.Props.ts

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+export enum ReadingMode {
+  /**
+   * Do not read the content at all.
+   */
+  DoNotRead = 0,
+
+  /**
+   * The content in this component will read out after other reading tasks of screen reader.
+   * E.g. first read the information of currently focus control, then this content.
+   * NOTE: ReadingMode.ReadAfterOtherContent doesn't support ChromeVOX v53.
+   */
+  ReadAfterOtherContent = 1,
+
+  /**
+   * Read immediately after this component is triggered rendering and updated.
+   * Most screen readers will announce the word 'alert' before or after the whole sentence.
+   */
+  ReadImmediately = 2
+}
+
+export interface IScreenReaderAlertProps extends React.Props<HTMLElement> {
+  /**
+   * Use readingMode to specify whether to read the alert, or read before/after normal screen reader content.
+   *
+   * @default ReadingMode.ReadAfterOtherContent
+   */
+  readingMode?: ReadingMode;
+
+  /**
+   * Use indicator to specifically tell screen reader this text should be read even when text is not changed.
+   * Especially in case of re-reading the same text or the text that is possibly the same with the old one.
+   *
+   * Change the indicator to read the text no matter the text is changed or not.
+   * Keep indicator the same value or simply not use it to read ONLY when the text is changed.
+   */
+  indicator?: number;
```

Comment:
The comment still doesn't make sense to me. I think this is a form of `itemVersion` which is a numerical value to "kick" the component into re-reading. Let me try to reword the comment:

In cases where you need the screen-reader to re-read the same comment again, you should pass in an incremented indicator value. For example, if an error had been read out, and the error occurs again due to some user input, you can increment the indicator number to have the message be re-read.

Does this sound right? Is this a good example? I'm struggling to understand the real use case. It also means the caller would need to store a numeric value in their state which is really awkward.